ledger/ledger_test.go

```diff
@@ -196,8 +203,10 @@ func TestLedgerBlockHeaders(t *testing.T) {
 	backlogPool := execpool.MakeBacklog(nil, 0, execpool.LowPriority, nil)
 	defer backlogPool.Shutdown()
 
-	initBlocks, initAccounts, _ := testGenerateInitState(t, protocol.ConsensusCurrentVersion)
-	l, err := OpenLedger(logging.Base(), t.Name(), true, initBlocks, initAccounts, crypto.Hash([]byte(t.Name())))
+	genesisInitState, _ := testGenerateInitState(t, protocol.ConsensusCurrentVersion)
+	const inMem = true
```

Comment:
nit: Your intuition to pass in variables here ( for readability ), it correct.
However, I would use constants for that, so that we won't need to define variables over and over.